docs/UserGuide.md

```diff
@@ -147,6 +156,19 @@ Clears all entries from the address book.
 
 Format: `clear`
 
+### Viewing a specific contact : `view`
+
+Views all the contact information of the specified contact.
+
+Format: `view t/TELEGRAM_HANDLE`
+
+* Displays all contact information of the person with specified `TELEGRAM_HANDLE`
+* `TELEGRAM_HANDLE` must be more than 4 characters, and can only contain letters, numbers and underscores.
```

Comment:
- should we say at least 5 characters instead? since we use the number 5 everywhere else. 
- maybe can mention the upper limit as well

Reply:
Okay, will change it to 5-32 characters